docs/products/kafka/concepts/upgrade-procedure.md

```diff
@@ -47,82 +46,103 @@ During an upgrade procedure:
+[Aiven Console](/docs/platform/howto/scale-services) or the
+[Aiven CLI](/docs/tools/cli/service-cli#avn-cli-service-update) to perform the downgrade.
 
-The upgrade procedure remains the same when changing the node type during a service plan
-change. If you downgrade to a service plan with nodes that have fewer CPUs, memory, or
-disk space, the latest system software versions are used for all newly created nodes.
-The upgrade mechanism, as explained in this document, is used to transfer data to
-the new nodes.
+The upgrade process remains the same when changing the node type during a service plan
+change. For instance, when downgrading to a plan with fewer resources, such as fewer
+CPUs, memory, or disk space, the latest system software versions are applied to all
+new nodes, and data is transferred accordingly.
 
 ## Upgrade impact and risks
 
-During the upgrade procedure, partition leadership coordination and streaming data to
-new nodes generate additional CPU load. To mitigate the risk, run the upgrade at a
-time of low traffic and reduce the normal workload on the cluster by disabling
-non-essential producers and consumers.
+Upgrading your cluster can increase CPU usage due to partition leadership coordination
+and data streaming to new nodes. To reduce the risk of disruptions, schedule the
+upgrade during low-traffic periods and minimize the cluster's normal workload by pausing non-essential producers and consumers.
 
-Specifically, when upgrading to a smaller plan, the disk can reach the
+If you are upgrading to a smaller plan, the disk may reach the
 [maximum allowed limit](https://aiven.io/docs/products/kafka/howto/prevent-full-disks),
-which can prevent the procedure from progressing. To mitigate the risk, check the disk
-usage before the upgrade and evaluate the amount of space left.
+which can block the upgrade. Check disk usage before upgrading and ensure there is enough free space.
 
 :::note
-In an emergency, our operations team can temporarily add additional volumes to
+In critical situations, Aiven's operations team can temporarily add extra storage to
 the old nodes.
 :::
+
+## Transitioning to KRaft
+
+With the release of Apache Kafka® 3.9, Aiven introduces support for Apache Kafka Raft
+(KRaft), the new consensus protocol for Kafka metadata management. This enhancement
+simplifies architecture while maintaining compatibility with existing features and
+integrations, including Aiven for Apache Kafka Connect, Aiven for Apache Kafka
+MirrorMaker 2, and Aiven for Karapace.
+
+All features and monitoring metrics supported in Apache Kafka 3.8 remain unchanged
+in APache Kafka 3.9, with no updates to ACL permissions or governance behaviors.
+
+### Availability and migration
+
+#### New services
+
+- All new Aiven for Apache Kafka services with Apache Kafka 3.9 will run KRaft as the
+  default metadata management protocol.
+- Initially supported on startup and business plans on Google Cloud Platform (GCP).
```

Comment:
There's a change in the plans

`startup-2` is not available in Kafka 3.9 and higher
`startup-4` is available in Kafka 3.9 and higher

Reply:
updated the content